recipe/meta.yaml

```diff
@@ -8,35 +7,29 @@ package:
 
 source:
   url: https://pypi.io/packages/source/s/setuptools/setuptools-{{ version }}.tar.gz
-  sha256: f10059f0152e0b7fb6b2edd77bcb1ecd4c9ed7048a826eb2d79f72fd2e6e237b
+  sha256: c3d4e2ab578fbf83775755cd76dae73627915a22832cf4ea5de895978767833b
   patches:
     # Modify setuptools to fail if used in conda build (encourage people to add all deps in meta.yaml).
     - patches/0002-disable-downloads-inside-conda-build.patch
     # distutils patches from python-feedstock
     - patches/0019-Use-ranlib-from-env-if-env-variable-is-set.patch
-    - patches/0008-runtime_library_dir_option-Use-1st-word-of-CC-as-com.patch
     - patches/0012-Disable-new-dtags-in-unixccompiler.py.patch
     - patches/0021-Add-d1trimfile-SRC_DIR-to-make-pdbs-more-relocatable.patch
-    - patches/0003-Win32-distutils-Add-support-to-cygwinccompiler-for-V.patch
-    - patches/0010-Add-support-for-_CONDA_PYTHON_SYSCONFIGDATA_NAME-if-.patch  # [py<=39 and python_impl=="cpython"]
     - patches/0006-Win32-Fixes-for-Windows-GCC-interop-needed-by-RPy2-a.patch
-    - patches/0034-distutils-fix-ldshared.patch
     # distutils patches from pypy3.6-feedstock
     - patches/0035-pypy-distutils-scheme.patch   # [win or py>37]
 
 build:
-  number: 1
+  number: 0
```

Comment:
We should skip `py<37` https://github.com/pypa/setuptools/blob/0ec53b228800300992ba1c53c2f089a435d4970c/setup.cfg#L27

Reply:
We've removed py36 and below from our build process, so pedantically: yes we should `skip: true  # [py<37]`; pragmatically we already do. Wouldn't it be just as good to remove these?

Reply:
I think it's for clarification if others want to know constraints about supported python versions

Reply:
https://github.com/pypa/setuptools/issues/2930

Reply:
Thanks for bringing this up and starting the discussion, @skupr-anaconda.